libpod/container.go

```diff
@@ -425,18 +425,6 @@ type ContainerConfig struct {
 	HealthCheckConfig *manifest.Schema2HealthConfig `json:"healthcheck"`
 }
 
-// ContainerNamedVolume is a named volume that will be mounted into the
-// container. Each named volume is a libpod Volume present in the state.
-type ContainerNamedVolume struct {
```

Comment:
We can't move this. It's a database struct.

We'll have to make a copy that we can use outside of Libpod.

Reply:
mheon, then we will need to change the struct in specgen to something else.